index.js

```diff
@@ -49,6 +49,7 @@ class MultiBuild {
    *    don't poison the cache for other targets.
    *  @param {Array<String>} [skipCache] - Names of targets that should not use rollup's cache, eg
    *    because they are processed differently than other targets. Defaults to [].
+   *  @param {String[]} [skipCacheModules] Module IDs to exclude from the cache.
```

Comment:
Seems that the name of this option is `skipResolveCache` below?

Reply:
haha yeah I caught that a minute later in 46e32b7a64d0ea6e27e7de261abfb6dc51752a8d.